.travis.yml

```diff
@@ -1,44 +1,51 @@
 jobs:
   include:
-  - name: Python 3.6
-    language: python
-    python: '3.6'
-    sudo: required
-    dist: xenial
-  - name: Python 3.7
-    language: python
-    python: '3.7'
-    sudo: required
-    dist: xenial
-  - name: Windows
-    language: c
-    python: '3.7'
-    os: windows
+    - name: Python 3.6
+      language: python
+      python: '3.6'
+      sudo: required
+      dist: xenial
+    - name: Python 3.7
+      language: python
+      python: '3.7'
+      sudo: required
+      dist: xenial
+    - name: Windows
+      language: c
+      python: '3.7'
+      os: windows
+
 install:
-- if [[ "$TRAVIS_OS_NAME" != "windows" ]]; then wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
-  -O miniconda.sh; export python_version=$TRAVIS_PYTHON_VERSION; bash miniconda.sh
-  -b -p $HOME/miniconda; source "$HOME/miniconda/etc/profile.d/conda.sh"; fi
-- if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then choco install miniconda3 --params="'/JustMe
-  /AddToPath:1'"; export PATH="/c/tools/miniconda3/:/c/tools/miniconda3/Scripts:/c/tools/miniconda3/Library/bin:$PATH";
-  source /c/tools/miniconda3/etc/profile.d/conda.sh; fi
-- conda config --set always_yes yes --set changeps1 no
-- conda update -q conda
-- bash scripts/install_deepchem_conda.sh deepchem
-- conda activate deepchem
-- python setup.py install
-- conda install mypy
-- pip install coveralls yapf==0.22.0
+  - if [[ "$TRAVIS_OS_NAME" != "windows" ]]; then
+      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh; 
+      export python_version=$TRAVIS_PYTHON_VERSION;
+      bash miniconda.sh -b -p $HOME/miniconda;
+      source "$HOME/miniconda/etc/profile.d/conda.sh";
+    fi
+  - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then 
+      choco install miniconda3 --params="'/JustMe /AddToPath:1'";
+      export PATH="/c/tools/miniconda3/:/c/tools/miniconda3/Scripts:/c/tools/miniconda3/Library/bin:$PATH";
+      source /c/tools/miniconda3/etc/profile.d/conda.sh;
+    fi
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+  - bash scripts/install_deepchem_conda.sh deepchem
+  - conda activate deepchem
+  - python setup.py install
+  - pip install coveralls mypy yapf==0.22.0
+
 script:
-- pytest -m "not slow" --cov=deepchem deepchem
-- if [ $TRAVIS_PYTHON_VERSION == '3.7' ]; then
-  find ./deepchem -name "*.py" ! -name '*load_dataset_template.py' | xargs python -m doctest -v; fi
-- bash devtools/travis-ci/test_format_code.sh
-- mypy -p deepchem --ignore-missing-imports
-- if [ $TRAVIS_PYTHON_VERSION == '3.7' ]; then
-  cd docs && pip install -r requirements.txt && make clean html && cd ..; fi
+  - pytest --cov=deepchem deepchem
+  - bash devtools/run_doctest.sh
+  - mypy -p deepchem
```

Comment:
You need to include the flag `--ignore-missing-imports`.  Otherwise it will report lots of errors from other packages we import that don't provide annotations.

Reply:
I move the `mypy` configuration to `setup.cfg`, so I think the option is not necessary

Reply:
Ok!